src/bci_build/package/__init__.py

```diff
@@ -868,6 +874,10 @@ def dockerfile_from_line(self) -> str:
     def kiwi_derived_from_entry(self) -> str:
         if self._from_image is None:
             return ""
+        # Adjust for the absolutely braindamaged format that OBS expects to reference
+        # external images
+        if self.is_base_container_annotation_available:
+            return f" derived_from=\"obs://SUSE:Registry/standard/{self._from_image.replace('registry.suse.com/','').replace(':', '#')}\""
```

Comment:
nit:  a bit more clear
```suggestion
            # cleanup image name
            image = self._from_image.replace('registry.suse.com/','').replace(':', '#')
            return f" derived_from=\"obs://SUSE:Registry/standard/{image}\""
```

Reply:
good suggestion, merged.